pom.xml

```diff
@@ -116,6 +116,9 @@
                     <links>
                         <link>http://joda-time.sourceforge.net/apidocs/</link>
                     </links>
+                    <excludePackageNames>
+                    	org.sagebionetworks.bridge.sdk.json,org.sagebionetworks.bridge.sdk.utils
```

Comment:
For some reason, Maven's javadoc plug-in doesn't recognize the second package if it's on its own line, which should be perfectly fine.